inc/namespace.php

```diff
@@ -167,6 +169,15 @@ function load_platform( $wp_debug_enabled ) {
 		load_object_cache_redis();
 	}
 
+	// Session support.
+	require_once __DIR__ . '/session_handler/namespace.php';
```

Comment:
For files that don't have side effects we add those to the module's `autoload.files` section in the `composer.json` rather than requiring directly. When testing locally you might need to commit the change before running `composer update` to get the updated autoload files.

Reply:
I was wondering why I didn't see any require calls for other files 🤦